client/scss/components/_promo.scss

```diff
@@ -20,11 +20,22 @@
     margin-bottom: $vertical-space-unit;
   }
 
+  &:hover .image,
+  &:focus .image {
+    transform: scale(1.2);
+  }
+
+  .image {
```

Comment:
Just making a note about what we spoke about:
It might be worth having a `image--full-width` and passing this into the `image` component.